### PR TITLE
Update PBL documents 

### PR DESCRIPTION
**Pull request by vickyiii**
1. Update the readme
2. add the x86simulation in Makefile
3. Change the fir single kernel code to drop the first output

### CODE REVIEW

**Review by mariodruiz — CHANGES_REQUESTED** [review]

File: docs/fir_single_kernel.md [hunk 1]
```diff
@@ -10,10 +10,30 @@ mermaid: true
...
-
+This lab provides the entire process of designing a low-pass filter using Python in a Jupyter notebook environment and deploying it to an AIE, showcasing the filtered results. This lab provides guidelines for coding such a filter in the Vitis™ AI Engine tools: for the experienced programmer it provides a structured approach to code low-order FIR filters. For learners it also provides background of key concepts such as AIE vectors, AIE APIs, and data flow. The following table shows the specific design steps in each part.
+
+| Part | Topic                                                                   | Steps                                                                                        |
+| ---- | ----------------------------------------------------------------------- | -------------------------------------------------------------------------------------------- |
```
[the remark this answers is not in the document]
Comment (vickyiii) [reply docs/fir_single_kernel.md:16]: Fixed

File: docs/fir_single_kernel.md [hunk 2]
```diff
@@ -10,10 +10,30 @@ mermaid: true
...
+| 1    | [Software Implementation](./notebook/fir_lowpass1.ipynb)                   | Generate the Complex Signal Data with High-frequency Noise                                   |
```
[the remark this answers is not in the document]
Comment (vickyiii) [reply docs/fir_single_kernel.md:17]: Done

File: docs/fir_single_kernel.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mariodruiz) [reply docs/fir_single_kernel.md:16]: I don't see the changes

File: pbl/aie_single_kernel/fir_lowpass/aie/Makefile [hunk 3]
```diff
@@ -76,7 +79,7 @@ get_output:
 
 x86sim:
```
[the remark this answers is not in the document]
Comment (vickyiii) [reply pbl/aie_single_kernel/fir_lowpass/aie/Makefile:80]: I prefer to set the aieemu and x86sim as two independent rules. To make the above change requires a big update on the tutorial.
https://github.com/Xilinx/Vitis-Tutorials/blob/714ef8d4a0b3ba1cc4d7ed993e1fe5ed5c4cd9c5/AI_Engine_Development/AIE/Feature_Tutorials/08-dsp-library/part_2/Makefile#L36

File: docs/fir_single_kernel.md [hunk 4]
```diff
@@ -94,17 +159,16 @@ Run the Software implementation in Jupyter lab on Windows or Linux system
    ```sh
    cd $HOME/pbl/aie_single_kernel/fir_lowpass/notebook
    py -m jupyter lab
-   ```
-
-2. Run the notebooks in Jupyter lab
+```
```
Comment (mariodruiz) [inline docs/fir_single_kernel.md:162]:
```suggestion
    ```
```

**Review by mariodruiz — APPROVED** [review]

**Pull request merged**